src/amo/pages/Home/index.js

```diff
@@ -254,11 +254,14 @@ export class HomeBase extends React.Component {
             }}
           />
 
-          {errorHandler.renderErrorIfPresent()}
+          {(!_config.get('enableFeatureHeroRecommendation') ||
+            clientApp === CLIENT_APP_ANDROID) &&
+            errorHandler.renderErrorIfPresent()}
```

Comment:
I suggest wrapping this so you can add the `padding: 0 24px` style to it directly:

```jsx
{errorHandler.hasError() 
  ? <div className="Home-noHeroError">{errorHandler.renderError()}</div>
  : null}
```